docs/src/main/asciidoc/kafka/kafka-streams.adoc

```diff
@@ -1807,13 +1807,24 @@ public Function<KStream<String, String>, KStream<String, String>> process(KafkaS
 }
 ```
 
+=== Metrics using Micrometer
+When Micrometer is on the classpath of a Spring Cloud Stream Kafka Streams binder based application, its consumer topic offset lags are collected as a gauge metric -
+`spring.cloud.stream.binder.kafka.offset`.
+
+The offset lag is computed for each consumed topic and partition whenever the mentioned metric is queried.
+When this computation fails (i.e. because of a timeout) a fallback value is used that is periodically calculated in the background.
+
+The metric collection behaviour can be configured by setting properties in the `spring.cloud.stream.kafka.binder.metrics` namespace,
+refer to the <<kafka-streams-binder-properties, properties section>> for more information.
+
 === Configuration Options
```

Comment:
You added these to the Kafka Streams binder docs. It should instead go into the regular Kafka binder [docs](https://github.com/spring-cloud/spring-cloud-stream/blob/main/docs/src/main/asciidoc/kafka/kafka_overview.adoc). Could you please move it there? Other than that, the PR looks good. Let me know when you are done with this change, and then we can merge.

Reply:
Ah what a slipup! I moved (part of) the documentation as the already present doc seemed sufficient enough, the property doc does the rest